runner.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
this file is just written vey terribly.
1) the way it is written, the expensive modeling part is executed every single time this file is imported. 
2) Some hyper parameters  that determine which data to use, what to name the output file, should all be input from the user.

Solutions,
1) wrap the executable part in 

if __name__ == '__main__':

code block,
this ensures that all the lines (not wrapped in functions, classes) are only run when you run something like python runner.py ......arguments.... from your terminal

2) AVOID HARD CODING. engineering principle 101. have them be input from the user instead. the standard thing to use here is to use python's builtin argparse module. This wraps stuff around the vanilla input(), raw_input() stuff and makes your life easier to write python executables. example code https://gist.github.com/riceluxs1t/60cb8509d9dc7bcecacf335120e3386c